daemon/config/config.go

```diff
@@ -121,6 +121,8 @@ type NetworkConfig struct {
 	DefaultAddressPools opts.PoolsOpt `json:"default-address-pools,omitempty"`
 	// NetworkControlPlaneMTU allows to specify the control plane MTU, this will allow to optimize the network use in some components
 	NetworkControlPlaneMTU int `json:"network-control-plane-mtu,omitempty"`
+	// Mtu allows to specify the MTU used for creating bridge networks if nothing else is specified upon creation
```

Comment:
Overlay networks could/should use this parameter too

```suggestion
	// Mtu allows to specify the MTU used for creating custom networks if nothing else is specified upon creation
```